packages/chakra-components/src/ComboBox/index.js

```diff
@@ -276,21 +290,24 @@ const ComboBoxInput = forwardRef(
   ========================================================================== */
 
 const ComboBoxSelectedGhost = forwardRef(({ size, ...props }, ref) => {
+  const { isListBox, isOpen, setOpen } = useComboBoxContext();
+
   const baseInputProps = inputSizes[size] && inputSizes[size];
 
   return (
     <Box
       display="flex"
       alignItems="center"
-      justifyContent="center"
       position="absolute"
       top="0"
       left="1px"
       zIndex={2}
       ref={ref}
-      pointerEvents="none"
+      width={isListBox ? "100%" : null}
```

Comment:
Should the null be auto ?

Reply:
Currently I am only focused on MultiSelect folder which has all the behaviours.
But to answer your question, setting the value to null means we are not providing any width property.
It will take it's parent's width.